### ISSUE TITLE
Allow dashboard subscriptions to sent only a selected tab of the dashboard

### ISSUE THREAD

**Issue by psalinasy** [metabase/metabase#36366]
**Is your feature request related to a problem? Please describe.**
When you have a dashboard with more than one tab dashboard subscriptions will treat the dashboard as a whole and you can not select with tab you want to include in the subscription. 

**Describe the solution you'd like**
Allow users to select a tab or tabs to be included in the dashboard subscription. 

**Describe alternatives you've considered**
An alternative could be to manage different dashboards or try to leverage filters. 

**How important is this feature to you?**

**Additional context**
Requested by a user.

**Comment by merritttravis49** [comment]
Can you add this to the roadmap? This is a feature that we need as well. It is cumbersome to have a recipient receive every tab's information in the body of the email when only one tab is relevant.

**Comment by UCJohnny** [comment]
Being able to select exactly which cards to include in the email would be even better than selecting full tabs.

**Comment by abhishek-superk** [comment]
We have org level and team level performance as different tabs in the same performance dashboard. 
Is sending every team's performance to all, or having so many dashboards for a single review the only two options?

**Comment by pawlodkowski** [comment]
Just chiming in here as well to kindly request this feature. I love Metabase subscriptions, especially for sending Slack alerts, but I agree that it's odd that there's no way to scope / limit the subscription to single tab of a dashboard.

My particular use case is that I want to send an hourly alert to Slack if two charts on a dashboard return results.
- And it works, but what happens then is that **all the text** across all tabs the dashboard get added to the body of the slack message as well, which is totally unnecessary.
- One workaround would be to put all the stuff that I don't want to be sent to slack in one tab, and then have a tab that has _just those two charts_, but this doesn't work because the subscription doesn't allow for tab-level granularity.

_EDIT_: my final workaround was to create multiple question-level alerts, which is fine for now, but this requires me to manually create an alert for each question which a bit more tedious (and harder to maintain).

**Comment by mkkangg** [comment]
Agreed on all of the previous comments. Customizing the subscription will be important to share only relevant information with different teams
